src/ui/settings/index.js

```diff
@@ -96,6 +96,15 @@ export default function setupSettings(kbnServer, server, config) {
     return setMany(changes);
   }
 
+  function setStatus() {
+    if (!kbnServer.config.get('plugins.initialize')) {
```

Comment:
We should probably move this check to the top of the mixin so that we don't expose the API or really do anything when we are disabled.